src/main/java/ua/javarush/constants/EnglishAlphabet.java

```diff
@@ -0,0 +1,18 @@
+package ua.javarush.constants;
+
+public class EnglishAlphabet {
+
+    public static final Character[] ENGLISH_ALPHABET = new Character[]{
+            'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'G', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U',
+            'V', 'W', 'X', 'Y', 'Z', 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'g', 'k', 'l', 'm', 'n', 'o', 'p',
+            'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z', '.', ',', '«', '»', '"', '\'', ':', '!', '?', ' '};
+
+//    public static final Character[] ENGLISH_ALPHABET = new Character[]{
```

Comment:
Do we need a commented code? If we need it please uncomment it, if no - remove it.